File: test/fixtures/output/node/unirest/nested.js
const unirest = require("unirest");

const req = unirest("GET", "http://mockbin.com/har");

req.query({
  "foo[bar]": "baz,zap",
  "fiz": "buz",
  "key": "value"
});

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/r/httr/fixtures/nested.r
library(httr)

url <- "http://mockbin.com/har"

queryString <- list(
  foo[bar] = "baz,zap",
  fiz = "buz"
)

response <- VERB("GET", url, query = queryString, content_type("application/octet-stream"))

content(response, "text")